suites/pacific/cephfs/tier_0_fs.yaml

```diff
@@ -65,6 +65,22 @@ tests:
                   label: mds
       destroy-cluster: false
       abort-on-fail: true
+  - test:
+      name: Add Active Active configuration of MDS
+      desc: Add Active Active configuration of MDS for cephfs
+      module: test_bootstrap.py
+      polarion-id:
```

Comment:
Could you add the Polarion-ID?

Reply:
and also make sure the TC is approved, marked as automated and you add the test case link in the automation link.

Reply:
Hi Rakesh,

TC is approved and marked as automated in polarion.